packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -381,11 +398,14 @@ class TransformLayer extends OffsetLayer {
 
   @override
   void addToScene(ui.SceneBuilder builder, Offset layerOffset) {
+    Timeline.startSync('TransformLayer::addToScene');
     Matrix4 offsetTransform = new Matrix4.identity();
-    offsetTransform.translate(offset.dx + layerOffset.dx, offset.dy + layerOffset.dy);
+    offsetTransform.translate(
+        offset.dx + layerOffset.dx, offset.dy + layerOffset.dy);
```

Comment:
style nit: this should either be one line, or the `);` should have a newline before it.